spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/AbstractHealthIndicator.java

```diff
@@ -31,13 +34,19 @@
  */
 public abstract class AbstractHealthIndicator implements HealthIndicator {
 
+	private static final Log logger = LogFactory.getLog(AbstractHealthIndicator.class);
+
 	@Override
 	public final Health health() {
 		Health.Builder builder = new Health.Builder();
 		try {
 			doHealthCheck(builder);
 		}
 		catch (Exception ex) {
+			logger.warn(String.format(
+					"%s failed: %s",
+					this.getClass().getSimpleName(),
+					ex.getMessage()));
```

Comment:
I think it'd be better to include the whole exception by using `warn(String, Throwable)` rather than just the message, particularly as the message is already part of the health's detail